etc/kayobe/environments/ci-multinode/stackhpc-monitoring.yml

```diff
@@ -1,3 +1,3 @@
 ---
 # Path to a CA certificate file to trust in the OpenStack Capacity exporter.
-stackhpc_os_capacity_openstack_cacert: "{{ kayobe_env_config_path }}/kolla/certificates/ca/vault.crt"
+stackhpc_os_capacity_openstack_cacert: "{{ kayobe_env_config_path }}/kolla/certificates/ca/openbao.crt"
```

Comment:
This is going to require some changes to the Multinode deployment script right? https://github.com/stackhpc/terraform-kayobe-multinode/blob/main/ansible/files/multinode.sh#L138